.github/workflows/scripts/unix_docker_build.sh

```diff
@@ -7,12 +7,20 @@ PY=$1
 GPU_BUILD=$2
 PROJECT_NAME=$3
 CI_SETUP_CMAKE_ARGS=$4
-
+export SCCACHE_DIR=/home/dev/sccache_cache
```

Comment:
Let's put sccache related setup code together? ;) Now it's split by conda & pip :P